`make test` failure on master, probably trivial fix

## Steps to reproduce the issue

`$ make test`

## Expected behavior

Should pass.

## Actual behavior

```
ERROR: Assertion failed: selection contains 12 elements instead of the asserted 11: t:SB_LUT4
Selection contains:
mux16/Y_SB_LUT4_O_I1_SB_LUT4_O_1_I1_SB_LUT4_O_2_I0_SB_LUT4_O_1
mux16/Y_SB_LUT4_O_I1_SB_LUT4_O
mux16/Y_SB_LUT4_O_I1_SB_LUT4_O_1_I1_SB_LUT4_O_1
mux16/Y_SB_LUT4_O_I1_SB_LUT4_O_1_I1_SB_LUT4_O_2_I0_SB_LUT4_O
mux16/Y_SB_LUT4_O_I1_SB_LUT4_O_I0_SB_LUT4_O_I3_SB_LUT4_O
mux16/Y_SB_LUT4_O_I1_SB_LUT4_O_1_I1_SB_LUT4_O
mux16/Y_SB_LUT4_O_I1_SB_LUT4_O_I0_SB_LUT4_O_2
mux16/Y_SB_LUT4_O
mux16/Y_SB_LUT4_O_I1_SB_LUT4_O_I0_SB_LUT4_O_1
mux16/Y_SB_LUT4_O_I1_SB_LUT4_O_I0_SB_LUT4_O
mux16/Y_SB_LUT4_O_I1_SB_LUT4_O_1_I1_SB_LUT4_O_2
mux16/Y_SB_LUT4_O_I1_SB_LUT4_O_1
make[1]: *** [run-test.mk:77: run-mux.ys] Error 1
make[1]: Leaving directory '/home/user/github/yosys/tests/arch/ice40'
```

Looks like it's also causing CI failures, e.g. for the rebased #2019 and #2093 .

--- a/tests/arch/ice40/mux.ys
+++ b/tests/arch/ice40/mux.ys
@@ -35,6 +35,7 @@ proc
 equiv_opt -assert -map +/ice40/cells_sim.v synth_ice40 # equivalency check
 design -load postopt # load the post-opt design (otherwise equiv_opt loads the pre-opt design)
 cd mux16 # Constrain all select calls below inside the top module
-select -assert-count 11 t:SB_LUT4
+select -assert-min 11 t:SB_LUT4
+select -assert-max 12 t:SB_LUT4
 
 select -assert-none t:SB_LUT4 %% t:* %D

